iteratee/src/main/scala/io/finch/iteratee/Enumerate.scala

```diff
@@ -0,0 +1,38 @@
+package io.finch.iteratee
+
+import java.nio.charset.Charset
+
+import com.twitter.io.Buf
+import com.twitter.util.Future
+import io.finch.Application
+import io.iteratee.Enumerator
+
+/**
+  * Enumerate HTTP streamed payload represented as [[Enumerator]] (encoded with [[Charset]]) into
+  * an [[Enumerator]] of arbitrary type `A`.
+  */
+trait Enumerate[A] {
+
+  type ContentType <: String
+
+  def apply(enumerator: Enumerator[Future, Buf], cs: Charset): Enumerator[Future, A]
+}
+
+object Enumerate extends EnumerateInstances {
+
+  type Aux[A, CT <: String] = Enumerate[A] {type ContentType = CT}

```

Comment:
@vkostyukov Not sure that it would help because this implicit is hidden behind `ToResponse` resolution, so it's gonna be another `implicitNoFound` (for `ToResponse`)